net/tinyproxy/Portfile

```diff
@@ -32,32 +31,48 @@ depends_build-append    port:asciidoc \
 configure.args          --disable-regexcheck \
                         --disable-silent-rules
 
+set conf_dir            ${prefix}/etc/${name}
+set log_dir             ${prefix}/var/log/${name}
+set run_dir             ${prefix}/var/run/${name}
+set doc_dir             ${prefix}/share/doc/${name}
+
+destroot.keepdirs       ${destroot}${conf_dir} \
+                        ${destroot}${log_dir} \
+                        ${destroot}${run_dir}
+
 post-destroot {
-    xinstall -d -o nobody -g nobody ${destroot}${prefix}/var/run/tinyproxy
-
-    move ${destroot}${prefix}/etc/tinyproxy/tinyproxy.conf ${destroot}${prefix}/etc/tinyproxy/tinyproxy.conf.default
-
-    set docdir ${destroot}${prefix}/share/doc/${name}
-    xinstall -d ${docdir}
-    xinstall -m 644 -W ${worksrcpath} \
-        AUTHORS \
-        ChangeLog \
-        COPYING \
-        docs/http-error-codes.txt \
-        docs/http-rfcs.txt \
-        README \
-        TODO \
-        ${docdir}
+    xinstall -d -o nobody -g nobody ${destroot}${log_dir}
+    move ${destroot}${conf_dir}/tinyproxy.conf ${destroot}${doc_dir}
+}
+
+post-activate {
+    if {![file exists ${conf_dir}/tinyproxy.conf]} {
+        copy ${doc_dir}/tinyproxy.conf ${conf_dir}/
+    }
 }
 
 startupitem.create      yes
 startupitem.netchange   yes
-startupitem.executable  ${prefix}/sbin/tinyproxy -c ${prefix}/etc/tinyproxy/tinyproxy.conf -d
+startupitem.executable  ${prefix}/bin/tinyproxy -d
```

Comment:
If it's really being run automatically, shouldn't it stay in sbin?

Reply:
It was changed in tinyproxy/tinyproxy#157.

Reply:
I guess it doesn't matter much.